sources/packages/backend/apps/api/src/services/education-program-offering/custom-validators/has-valid-offering-period-for-funded-days.ts

```diff
@@ -20,32 +20,32 @@ class HasValidOfferingPeriodForFundedDaysConstraint
   implements ValidatorConstraintInterface
 {
```

Comment:
The constraint name(also the file name) `HasValidOfferingPeriodForFundedDaysConstraint` is not accurate anymore IMO.  The validation is not just about funded days anymore. It is based on funded days and weeks.